### ISSUE TITLE
question

### ISSUE THREAD

**Issue by superCman** [nerox8664/pytorch2keras#47]
please，whether batchnorm1d is supported？？
